projects/ion/src/lib/checkbox/checkbox.component.ts

```diff
@@ -0,0 +1,37 @@
+import { CommonModule, NgClass } from '@angular/common';
+import {
+  ChangeDetectionStrategy,
+  Component,
+  input,
+  model,
+  output,
+} from '@angular/core';
+import { IonCheckboxProps } from './types';
+
+@Component({
+  standalone: true,
+  selector: 'ion-checkbox',
+  imports: [CommonModule, NgClass],
+  templateUrl: './checkbox.component.html',
+  styleUrls: ['./checkbox.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class IonCheckboxComponent {
+  label = input<IonCheckboxProps['label']>();
+  value = input<IonCheckboxProps['value']>();
```

Comment:
```suggestion
  value = input.required<IonCheckboxProps['value']>();
```
I think this property should be required